app/src/main/java/com/github/houseorganizer/houseorganizer/panels/MainScreenActivity.java

```diff
@@ -108,7 +108,7 @@ protected void onResume() {
                     FirebaseStorage storage = FirebaseStorage.getInstance();
                     // this creates the reference to the picture
                     StorageReference imageRef = storage.getReference().child(calendarAdapter.getEventToAttach() + ".jpg");
```

Comment:
So for now uploaded images have to be jpg ? I mean it does not restrict a lot on phones but just to be sure

Reply:
Im pretty sure it can automatically go from png/jpeg/etc to jpg anyways, need to test it though